foundations/azure/referenceImplementations/core/managementGroupTemplates/policyDefinitions/Compliant-StoragePolicySetDefinition.json

```diff
@@ -56,6 +56,795 @@
+                        "metadata": {
+                            "version": "1.0.0",
+                            "category": "Storage"
+                        },
+                        "parameters": {
+                            "effect": {
+                                "type": "String",
+                                "metadata": {
+                                    "displayName": "Effect",
+                                    "description": "Enable or disable the execution of the policy"
+                                },
+                                "allowedValues": [
+                                    "Deny",
+                                    "Audit",
+                                    "Disabled"
+                                ],
+                                "defaultValue": "Deny"
+                            }
+                        },
+                        "policyRule": {
+                            "if": {
+                                "allOf": [
+                                    {
+                                        "field": "type",
+                                        "equals": "Microsoft.Storage/storageAccounts/privateEndpointConnections"
+                                    },
+                                    {
+                                        "field": "Microsoft.Storage/storageAccounts/privateEndpointConnections/privateLinkServiceConnectionState.status",
+                                        "equals": "Approved"
+                                    },
+                                    {
+                                        "anyOf": [
+                                            {
+                                                "field": "Microsoft.Storage/storageAccounts/privateEndpointConnections/privateEndpoint.id",
+                                                "exists": false
+                                            },
+                                            {
+                                                "value": "[[split(concat(field('Microsoft.Storage/storageAccounts/privateEndpointConnections/privateEndpoint.id'), '//'), '/')[2]]",
+                                                "notEquals": "[[subscription().subscriptionId]"
+                                            }
+                                        ]
+                                    }
+                                ]
+                            },
+                            "then": {
+                                "effect": "[[parameters('effect')]"
+                            }
+                        }
+                    },
+                    "name": "Deny-Storage-PrivateEndpointConnections"
+                },
+                {
+                    "properties": {
+                        "displayName": "Public blob access should be restricted for Storage Accounts",
```

Comment:
This seems redundant to /providers/Microsoft.Authorization/policyDefinitions/b2982f36-99f2-4db5-8eff-283140c09693 (Storage accounts should disable public network access) which we are including, no?

Reply:
The policy you mentioned cannot be used in a data context when relying on Databricks or Synapse. For these services, we have to rely on the Resource access rules feature, which is why I included all the other necessary policies to keep the environment as secure as possible.

Reply:
Do you want me to remove all the related policy rules for now?